shongo-controller/src/main/java/cz/cesnet/shongo/controller/rest/ClientWebUrl.java

```diff
@@ -0,0 +1,58 @@
+package cz.cesnet.shongo.controller.rest;
+
+/**
+ * Web URL paths for REST endpoints.
+ *
+ * @author Filip Karnis
+ */
+public class ClientWebUrl
```

Comment:
By som zmenil na RestUrl, alebo niečo iné.

Reply:
zmenene na RestApiPath